### PR TITLE
Protocol v0.2

### PR DESCRIPTION
**Pull request by kry127**

### CODE REVIEW

File: proto/api/v0_2/sink/sink_service.proto [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import "api/v0_2/sink/control.proto";
+
+service SourceService {
+    // Spec is method for determining endpoint settings available for configuring
+    rpc Spec(SpecReq) returns (SpecRsp) {}
+    // Check is method that checks settings of endpoint are comply to specification
+    // In order to see specification see Spec handle
+    rpc Check(CheckReq) returns (CheckRsp) {}
+
+    rpc Write(stream WriteReq) returns (stream WriteRsp) {}
```
Comment (laskoviymishka) [inline proto/api/v0_2/sink/sink_service.proto:15]: SourceService should be readonly

File: proto/api/v0_2/sink/sink_service.proto [hunk 2]
```diff
@@ -0,0 +1,26 @@
+syntax="proto3";
+
+package yandexcloud.datatransfer.dtextension.v0_2.sink;
+
+import "api/v0_2/common.proto";
+import "api/v0_2/sink/control.proto";
+
+service SourceService {
```
Comment (laskoviymishka) [inline proto/api/v0_2/sink/sink_service.proto]: mb SinkService?

**Comment by laskoviymishka** [comment]
need high level overview of data flow

File: proto/api/v0_2/sink/sink_service.proto (hunk 2 above)
Comment (kry127) [reply proto/api/v0_2/sink/sink_service.proto]: Yes

File: proto/api/v0_2/sink/sink_service.proto (hunk 1 above)
Comment (kry127) [reply proto/api/v0_2/sink/sink_service.proto:15]: You mean SinkService? It can be readonly indeed

**Comment by kry127** [comment]
> need high level overview of data flow

Added README.MD to source and sink which gives an instruction on dataflow implementation.

**Pull request merged**